ClientRuntimes/CSharp/ClientRuntime/Microsoft.Rest.ClientRuntime.nuget.proj

```diff
@@ -6,7 +6,8 @@
     -->
     <SdkNuGetPackage Include="Microsoft.Rest.ClientRuntime">
       <PackageVersion>1.1.0</PackageVersion>
-      <Folder>$(MSBuildThisFileDirectory)</Folder>
+      <Folder>ClientRuntimes/CSharp/ClientRuntime/</Folder>
```

Comment:
you can log an issue to trace to bring back the  usage of "MSBuildThisFileDirectory"